### DISCUSSION TITLE
.bbmodel support (Blockbench)

### DISCUSSION THREAD

**Discussion by tsikavo** [godotengine/godot-proposals#10128] [3D]
Not long ago, support for Blender models was added to the engine, but I use the simpler Blockbench more often. I'm wondering how popular it is among game developers on Godot, and how useful it would be to support this model in the engine. Blockbench is open source, supported by the community, and has a "readable" file format.
https://github.com/JannisX11/blockbench

**Comment by Calinou** [comment]
As Blockbench's README says:

> Models can be exported into standardized formats, to be shared, rendered, 3D-printed, or used in game engines.

If Blockbench has a CLI that allows automating exports, then it would be possible to have a Blockbench interface similar to Blender, but I think this is better left to an add-on since Blockbench isn't quite as popular as Blender.
